README.md

```diff
@@ -1,7 +1,7 @@
 [![CI Status](https://github.com/AY2021S1-CS2103T-W15-2/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103T-W15-2/tp/actions)
 
 ## Reeve
-![Ui](docs/images/Ui.png)
+![Ui](docs/images/ReeveGUI.png)
```

Comment:
What is the difference between ReeveGUI.png and Ui.png? Do we even need the former?

Reply:
Deleted ReeveGUI.png and replaced all occurrences with Ui.png thks